sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/catalog/interface.scala

```diff
@@ -693,4 +694,29 @@ case class HiveTableRelation(
   override def newInstance(): HiveTableRelation = copy(
     dataCols = dataCols.map(_.newInstance()),
     partitionCols = partitionCols.map(_.newInstance()))
+
+  override def simpleString(maxFields: Int): String = {
+    val catalogTable = tableMeta.storage.serde match {
+      case Some(serde) => tableMeta.identifier :: serde :: Nil
+      case _ => tableMeta.identifier :: Nil
+    }
+
+    val metadataEntries = Map(
+      "CatalogTable" -> catalogTable.mkString(", "),
+      "Data Cols" -> truncatedString(dataCols, "[", ", ", "]", maxFields),
+      "Partition Cols" -> truncatedString(partitionCols, "[", ", ", "]", maxFields),
+      "Statistic" -> tableStats.map(_.simpleString).getOrElse(""),
+      "Pruned Partitions" -> prunedPartitions.map { partitions =>
```

Comment:
AFAIK, in Hive each partition can have a different serde. Shall we display partition serde like before?

Reply:
> AFAIK, in Hive each partition can have a different serde. Shall we display partition serde like before?

Since table level `serde` wan shown, show partition's `serde` is ok for me.

But be honest, default meta value length is 100, one partition `serde` length always more than 30. It won't show too much info....

Reply:
maybe omit the partition serde if it's the same as table serde?

Reply:
> maybe omit the partition serde if it's the same as table serde?

Looks good to me. update later

Reply:
> maybe omit the partition serde if it's the same as table serde?

Strictly speaking, the field information for the partition may also be different

Reply:
> maybe omit the partition serde if it's the same as table serde?

Updated